planet/src/main/webapp/js/search-bar.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function getSearchResults() {
+    const resultsContainer = document.getElementById('search-results-container');
+    resultsContainer.innerHTML = '';
+    const searchInput = document.getElementById('search-input').value;
+    if (searchInput === '') {
+        return;
+    }
+    database.ref('users').orderByChild('name')  // Order elements by name
```

Comment:
do we already have an index on `name`? If not, it might be worth adding it

Reply:
It tried to but I couldn't since I already have an indexOn rule for the username.

Reply:
(Ignore this if you've already tried it.)

It looks like you can have multiple indexes for a given table
https://firebase.google.com/docs/database/security/indexing-data?hl=en
```
{
  "rules": {
    "dinosaurs": {
      ".indexOn": ["height", "length"]
    }
  }
}
```

Reply:
Haha it looks like it's working! Done! Thank you